### ISSUE TITLE
Concordion 2 with parallel run extension makes parent Spec not aggregate test results

### ISSUE THREAD

**Issue by ghost** [concordion/concordion-parallel-run-extension#2]
Our project is running tests in parallel using this extension; we have one main Spec which just links (through c:run) with the specs performing the real tests.  When we launch the main/parent Spec it is marking the execution as successful; its because the failures in child tests aren't being aggregated in the main parent. It just happens with 2.0.0

If you need code I might provide it here, but it happens with the simplest project you can build.

**Comment by nigelcharman** [comment]
Thanks for reporting this. It will need an update to Concordion core to fix. I've raised a PR for the change and will look to create a 2.0.1 release in the next few days.

**Comment by nigelcharman** [comment]
Please confirm this is fixed for you with Concordion 2.0.1. thanks

**Comment by ghost** [comment]
Hi Nigel,

I detected something weird: As the log for the parent spec now shows correct failures. Ex:

Successes: 0, Failures: 1

The JUnit runner shows that there were no errors and shows something like this on eclipse.

![image](https://cloud.githubusercontent.com/assets/10096558/14780518/7636423c-0add-11e6-96f7-8754aa134aa5.png)

This happens with the bare project and our big project aswell.

Our jenkins job also reports that it was successful.

**Comment by nigelcharman** [comment]
Thanks @amkam, I've reproduced it. 

On a long running test, you can see that the [Concordion Specification ...] test succeeds straight away without waiting for the test to finish.

Would you mind raising a new issue for this please?
